cric-predict/src/Components/Groups.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import '../Stylesheets/App.css';
+import '../Stylesheets/Group.css';
+import {Button}  from 'react-bootstrap';
+import PublicGroups from './PublicGroups';
+import UserGroups from './UserGroups';
+import CreateGroup from './CreateGroup';
+import {Redirect} from "react-router-dom";
+
+function GroupComponent() {
+      if(!window.localStorage.getItem('access_token')) {
```

Comment:
maybe we can explore a third-party plugin that will handle authentication and redirection on our behalf

Reply:
Okay